### ISSUE TITLE
Image Flags - Security Certificate Report over https

### ISSUE THREAD

**Issue by cristianizzo** [kondi0/ng4-intl-phone#27]
Please could you replace the cloudinary url from 

src="https://res.cloudinary.com/dvbuhh0bl/image/upload/v1495279723/flags_boi3c5.png"
to 
src="//res.cloudinary.com/dvbuhh0bl/image/upload/v1495279723/flags_boi3c5.png"

it will fix the issue.

**Comment by datcuemil** [comment]
I agree. it would be useful for that to be implemented. if the pull request is not accepted, i will fork the repo and do it myself :D

**Comment by hatemalhassani** [comment]
in our website we have Content Security Policy (CSP) header not allowed to load external files, could you please make it as local file.

**Comment by datcuemil** [comment]
@hatemalhassani I don't think there's anybody taking care of this repo. Best chance you have is to do like myself, fork it and do the changes you need.

**Comment by oleksandrsamoienko** [comment]
Workaround: 

Add to your component css:

int-phone-prefix/deep/ .flag {
  background: url('https://res.cloudinary.com/dvbuhh0bl/image/upload/v1495279723/flags_boi3c5.png') no-repeat;
}

int-phone-prefix/deep/ .defaultCountry {
  background: url('https://res.cloudinary.com/dvbuhh0bl/image/upload/c_scale,h_15,w_15/v1495279723/default_tmey2r.png') no-repeat;
}
